enterprise/api/v1/serializers.py

```diff
@@ -591,11 +591,53 @@ class Meta:
         write_only=True
     )
     enterprise_customer_catalogs = EnterpriseCustomerCatalogSerializer(many=True, read_only=True)
+    enterprise_catalogs = serializers.ListField(write_only=True, child=serializers.UUIDField(), default=[])
```

Comment:
To avoid confusing it with the already existing variable in the serializer `enterprise_customer_catalogs`, this variable name `enterprise_catalogs` could be changed to something a bit more specific. What exactly does this  return?

Reply:
@sameenfatima78 I have renamed the field name to `enterprise_customer_catalog_uuids` as it will contain UUIDs if of the enterprise customer catalog. Good catch 👍 previous name was very ambiguous.